runtimes/spiritnet/src/lib.rs

```diff
@@ -159,7 +169,7 @@ impl frame_system::Config for Runtime {
 }
 
 parameter_types! {
-	pub const MinimumPeriod: u64 = constants::SLOT_DURATION / 2;
+	pub const MinimumPeriod: u64 = constants::SLOT_DURATION.saturating_div(2);
```

Comment:
We don't want to saturate. We want to fail. This would happen at compilation time, so it's fine to fail here. Not failing means we would not catch any possible configuration issues.